unit-11/main.js

```diff
@@ -0,0 +1,70 @@
+// Exercise 1
+function makeLoopArray1(duplicate, number) {
```

Comment:
1. Thêm dấu cách vào
```suggestion
function makeLoopArray1 (duplicate, number) {
```

2. Đặt tên hàm sát nghĩa 1 tí, vd:
```suggestion
function repeatWord (word, times) {
```
```suggestion
function dublicateWord (word, times) {
```